dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -704,6 +704,10 @@ error.missing.version.files=CWWKE0952E: Version information cannot be read from
 error.missing.version.files.explanation=The directory that contains version information files either does not exist or cannot be read by the current user.
 error.missing.version.files.useraction=Ensure that the directory exists and can be read by the current user. 
 
+warning.earlyRelease=CWWKE0953W: This is an unsupported early release version of WebSphere Liberty.
```

Comment:
Should it say "WebSphere Liberty"? It might be Open Liberty? Is there a way to dynamically put the product name in? There must be some precedent.

Reply:
Message is changed to:    This version of {0} is an unsupported early release version.

The way I have it now it gets the product name from the lib/versions/xxxx.properties files.   If there is an openliberty.properties and a WebSphereApplicatonServer.properties and both are set to EARLY_ACCESS, then you get 2 messages:

Starting server defaultServer.
CWWKE0953W: This version of Open Liberty is an unsupported early release version.
CWWKE0953W: This version of WebSphere Application Server is an unsupported early release version.
Server defaultServer started with process ID 66213.

I don't know if both properties files always agree on whether it is an EARLY_ACCESS edition.

Reply:
I'm sure they do both use EARLT_ACCESS... if not that's a different bug!

Reply:
I don't think it matters if the message is output twice on WebSphere Liberty ... this is a beta anyway